src/view/components/cpx/Svg_utils.tsx

```diff
@@ -3,6 +3,8 @@
 
 // Adapted from : https://github.com/microsoft/pxt/blob/master/pxtsim/svg.ts
 
+/* tslint:disable */
```

Comment:
What was the reason to add this?

Reply:
That file is currently failing the tslint, but I will fix it

Reply:
(
unrelated to this pr)